change/@fluentui-react-dialog-b286942f-21da-4936-947b-6daf42db80ab.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 46 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27248&commitId=fd64fde648b956392efeaaab773c620da4c6e5b7&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1857 screenshots unchanged, 0 screenshots with different dimensions and 46 screenshots with visible difference.


  <details><summary><b>unknown</b> 46 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.Fluid Actions End.chromium.png|8344| Changed|
|Dialog.Fluid Actions Start.chromium.png|8345| Changed|
|Dialog.actions position start & position end - Dark Mode.chromium.png|7084| Changed|
|Dialog.actions position start & position end - High Contrast.chromium.png|10983| Changed|
|Dialog.actions position start & position end - RTL.chromium.png|7794| Changed|
|Dialog.actions position start & position end.chromium.png|7777| Changed|
|Dialog.actions position start - Dark Mode.chromium.png|6706| Changed|
|Dialog.actions position start - High Contrast.chromium.png|10219| Changed|
|Dialog.actions position start - RTL.chromium.png|7433| Changed|
|Dialog.actions position start.chromium.png|7417| Changed|
|Dialog.alert - Dark Mode.chromium.png|6705| Changed|
|Dialog.alert - High Contrast.chromium.png|10219| Changed|
|Dialog.alert - RTL.chromium.png|7434| Changed|
|Dialog.alert.chromium.png|7417| Changed|
|Dialog.default - Dark Mode.chromium.png|6705| Changed|
|Dialog.default - High Contrast.chromium.png|10219| Changed|
|Dialog.default - RTL.chromium.png|7434| Changed|
|Dialog.default.chromium.png|7417| Changed|
|Dialog.nested - Dark Mode.chromium.png|1642| Changed|
|Dialog.nested - High Contrast.chromium.png|1964| Changed|
|Dialog.nested - RTL.chromium.png|1692| Changed|
|Dialog.nested.chromium.png|1690| Changed|
|Dialog.no actions - Dark Mode.chromium.png|6144| Changed|
|Dialog.no actions - High Contrast.chromium.png|8811| Changed|
|Dialog.no actions - RTL.chromium.png|6665| Changed|
|Dialog.no actions.chromium.png|6649| Changed|
|Dialog.no title & no actions - Dark Mode.chromium.png|5751| Changed|
|Dialog.no title & no actions - High Contrast.chromium.png|8341| Changed|
|Dialog.no title & no actions - RTL.chromium.png|6263| Changed|
|Dialog.no title & no actions.chromium.png|6245| Changed|
|Dialog.no title - Dark Mode.chromium.png|6312| Changed|
|Dialog.no title - High Contrast.chromium.png|9749| Changed|
|Dialog.no title - RTL.chromium.png|7031| Changed|
|Dialog.no title.chromium.png|7014| Changed|
|Dialog.non-modal - Dark Mode.chromium.png|7558| Changed|
|Dialog.non-modal - High Contrast.chromium.png|10325| Changed|
|Dialog.non-modal - RTL.chromium.png|6659| Changed|
|Dialog.non-modal.chromium.png|6642| Changed|
|Dialog.scroll long content - Dark Mode.chromium.png|70976| Changed|
|Dialog.scroll long content - High Contrast.chromium.png|84893| Changed|
|Dialog.scroll long content - RTL.chromium.png|72300| Changed|
|Dialog.scroll long content.chromium.png|73032| Changed|
|Dialog.title custom action - Dark Mode.chromium.png|6654| Changed|
|Dialog.title custom action - High Contrast.chromium.png|9444| Changed|
|Dialog.title custom action - RTL.chromium.png|7216| Changed|
|Dialog.title custom action.chromium.png|7179| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>